enhancing/enhancer.spec.js

```diff
@@ -1,2 +1,62 @@
-const enhancer = require('./enhancer.js');
+const { succeed, fail, repair, get } = require("./enhancer.js");
 // test away!
+
+const item = { name: "Very Real Item", durability: 59, enhancement: 4 };
+
```

Comment:
good job testing your functions, I like how you added multiple describes within this describe to better highlight the purpose of each test.